sdk/dotnet/Pulumi/PublicAPI.Shipped.txt

```diff
@@ -322,8 +323,10 @@ static Pulumi.InputMap<V>.implicit operator Pulumi.InputMap<V>(Pulumi.Output<Sys
 static Pulumi.InputMap<V>.implicit operator Pulumi.InputMap<V>(Pulumi.Output<System.Collections.Immutable.ImmutableDictionary<string, V>> values) -> Pulumi.InputMap<V>
 static Pulumi.InputMap<V>.implicit operator Pulumi.InputMap<V>(System.Collections.Generic.Dictionary<string, V> values) -> Pulumi.InputMap<V>
 static Pulumi.InputMap<V>.implicit operator Pulumi.InputMap<V>(System.Collections.Immutable.ImmutableDictionary<string, V> values) -> Pulumi.InputMap<V>
+static Pulumi.InputUnion<T0, T1>.implicit operator Pulumi.InputUnion<T0, T1>(Pulumi.Output<Pulumi.Union<T0, T1>> value) -> Pulumi.InputUnion<T0, T1>
 static Pulumi.InputUnion<T0, T1>.implicit operator Pulumi.InputUnion<T0, T1>(Pulumi.Output<T0> value) -> Pulumi.InputUnion<T0, T1>
 static Pulumi.InputUnion<T0, T1>.implicit operator Pulumi.InputUnion<T0, T1>(Pulumi.Output<T1> value) -> Pulumi.InputUnion<T0, T1>
+static Pulumi.InputUnion<T0, T1>.implicit operator Pulumi.InputUnion<T0, T1>(Pulumi.Union<T0, T1> value) -> Pulumi.InputUnion<T0, T1>
```

Comment:
Would it make sense to add quick unit tests for these new APIs?

Reply:
Yes, I'll add some